docker-cpp-demos/rtsp-docker/start_rtsp_in_docker.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Demo GStreamer Sample Application for RTSP Streaming to Kinesis Video Streams
+# To be run inside the Docker container
+
+cd /opt/amazon-kinesis-video-streams-producer-sdk-cpp/build/
```

Comment:
You don't have to cd since the last `WORKDIR` is already set to the right dir, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/pull/6/files#diff-ff42ca0f8e06d8a0ccc07bd5096548faR37